### ISSUE TITLE
module eval should be `eval(m, expr)` instead of `eval(expr)` in `seval`

### ISSUE THREAD

**Issue by Roger-luo** [cjdoris/PythonCall.jl#178]
If you create new Julia module

```python
import juliacall as jl

m = jl.newmodule("TestModule")
m.seval("1 + 1")
```

will not work because `m` doesn't have `eval(::Expr)` defined, but only have `eval(::Module, ::Expr)` this is because the module created using `Module` doesn't have `eval(::Expr)` defined. I'm not sure if we should just generate this method manually, but I think if we use `Base.eval(::Module, ::Expr)` it will always work at least.

**Comment by cjdoris** [comment]
Thanks, this is already fixed on main (#170).

**Comment by Roger-luo** [comment]
Thanks! I didn't notice that PR, would it be possible to tag release soon? @cjdoris
